Jenkinsfile

```diff
@@ -10,116 +14,186 @@ pipeline {
     buildDiscarder(logRotator(numToKeepStr: '5', artifactNumToKeepStr: '5'))
     timestamps()
   }
+  parameters {
+    string(name: 'AWSRegion', defaultValue: 'us-east-1', description: 'Enter the desired AWS region')
+    string(name: 'ECRURI', defaultValue: '054017840000.dkr.ecr.us-east-1.amazonaws.com', description: 'Enter the URI of the Container Registry')
+    string(name: 'Email', defaultValue: 'vecinomio@gmail.com', description: 'Enter the desired Email for the Job notifications')
+    booleanParam(name: 'Build', defaultValue: true, description: 'Includes Build app and Tests')
+    booleanParam(name: 'Release', defaultValue: false, description: 'Includes Tagging and Delivery')
+    booleanParam(name: 'Deployment', defaultValue: false, description: 'Deploy a new version of App')
+    booleanParam(name: 'SetNewTag', defaultValue: false, description: 'Auto-increasing version')
+    choice(name: 'AppVersion', choices: ['Minor', 'Middle', 'Major'], description: 'Pick Version Tag')
```

Comment:
is this parameter used?

Reply:
Yes, it used. 
When developers, for example, want to increase middle or major part.

Reply:
it's confusing with `SetNewTag`

Reply:
Reconfigured according to the agreement.